src/experiment/trials/camera.js

```diff
@@ -9,111 +9,105 @@ const WEBCAM_ID = "webcam";
 
 /**
  * A trial that begins recording the participant using their computer's default camera
- * @param {Object} jsPsych The jsPsych instance being used to run the task
+ *
  * @returns {Object} A jsPsych trial object
  */
 // TODO @brown-ccv #301: Use jsPsych extension, deprecate this function
```

Comment:
Since this is no longer a function, more docstring updates. Also, I would have suggested renaming the variable, but since it looks like it's going to be deprecated, I think it's fine to leave as is

```suggestion
 *
 * @type {Object} A jsPsych trial object
 */
// TODO @brown-ccv #301: Use jsPsych extension, deprecate this variable
```